README.md

```diff
@@ -63,13 +65,17 @@ In the Arguments section, add five (or more) Arguments Passed On Launch to the b
   ```
   --base-path="/Users/YourName/Desktop/YourProjectFolder"
   ```
-- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs:  
+- The path within your project to your asset catalog, without a preceding slash (add multiple times if you have multiple asset catalogs):  
   ```
   --asset-catalog="Resources/Images.xcassets"
   ```  
   ```
   --asset-catalog="Resources/Media.xcassets"
   ```
+- The prefix for the category method names. If not specified, the default is `ac`. Must be at least 2 characters, and conform to valid method naming rules:
```

Comment:
Ok, but... how do we apply the prefixes for each usage? For example, in our project (which I think is following pretty standard convention), we prefix our category methods with `xyz_`, but our category names are prefixed with `XYZ`. So, our categories would look like:

`UIImage+XYZAssetCatalog.h/m` and
`+ (UIImage *)xyz_SomeImage;`

Would we accept only one prefix and mutate it to match these usages, forcing this convention? Or do we allow separate prefixes for methods and categories names to be specified?